examples/command/portals/kafka/aiven/run.sh

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env bash
+set -e
+
+# This script, `./run.sh ...` is invoked on a developer’s work machine.
+#
+# This hands-on example uses Ockam to create an end-to-end encrypted portal to Aiven Kafka Cloud.
+# We connect a kafka client in one virtual private network with a Kafka event streamer
+# in another virtual private network.
+#
+# The example uses docker and docker compose to create these virtual networks.
+#
+# You can read a detailed walkthrough of this example at:
+# https://docs.ockam.io/portals/kafka/aiven/docker
+
+run() {
+  # Run `ockam enroll`.
+  #
+  # The enroll command creates a new vault and generates a cryptographic identity with
+  # private keys stored in that vault. It then guides you to sign in to Ockam Orchestrator.
+  #
+  # If this is your first time signing in, the Orchestrator creates a new dedicated project
+  # for you. A project offers two services: a membership authority and a relay service.
+  #
+  # The enroll command then asks this project’s membership authority to sign and issue
+  # a credential that attests that your identifier is a member of this project. Since your
+  # account in Orchestrator is the creator and hence first administrator on this new project,
+  # the membership authority issues this credential. The enroll command stores the
+  # credential for later use and exits.
+  ockam enroll
+
+  # Creates a Kafka cluster and it's API key and secret using Aiven CLI
+  random_char=$(echo $RANDOM | md5sum | head -c 10)
+
+  service_name="ockam-demo-${random_char}"
+  avn service create "$service_name" -t kafka -p startup-2 --cloud google-europe-west1
```

Comment:
Is there a particular reason to use `google-europe-west1`? As `aws-us-east-1` is a common cloudprovider/region to use for our examples.

Reply:
Done thanks.